### ISSUE TITLE
Option to install last tag?

### ISSUE THREAD

**Issue by zilti** [dimitri/el-get#2482]
Is there currently a way to have El-Get install the checkout with the latest tag, thus mimicking MELPA Stable? There's e.g. CIDER which I want to have stable releases only.

**Comment by npostavs** [comment]
This is currently not supported. Patches welcome.

**Comment by zilti** [comment]
I'm thinking about implementing a `:checkout 'latest-tag` which does that, would this be ok, or do you favour another way of denoting that?

**Comment by npostavs** [comment]
Yeah, that sounds good. A possible variation would be `:checkout #'el-get-latest-tag` where `el-get-latest-tag` is the name of the function which returns the tag. This has the advantage that it's easy to substitute other functions with different policies (e.g., `el-get-latest-nonbeta-tag` or something), though it's a bit more verbose. If you implement this, you can decide which way you like best :)

**Comment by zilti** [comment]
I went with your idea of adding a function. Though I'm not really sure to what file I should add this function?

**Comment by npostavs** [comment]
`el-get-recipes.el` perhaps?
